src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1844,6 +1858,15 @@ private bool TryExtractStringAndIntValueFromAttribute(CustomAttributeHandle hand
             return result;
         }
 
+        private bool TryExtractStringAndStringValueFromAttribute(CustomAttributeHandle handle, out string? string1Value, out string? string2Value)
+        {
+            (string?, string?) data;
+            var result = TryExtractValueFromAttribute(handle, out data, s_attributeStringAndStringValueExtractor);
+            string1Value = data.Item1;
+            string2Value = data.Item2;
```

Comment:
nit: consider using a deconstruction assignment
```suggestion
            (string1Value, string2Value) = data;
```